cmd/consoleplugin/static/exposed-pendo-chunk-0f1e4648dfea45bc1052.min.js

```diff
@@ -0,0 +1 @@
+"use strict";(self.webpackChunkconsole_plugin_template=self.webpackChunkconsole_plugin_template||[]).push([[484],{248:(n,t,e)=>{e.r(t),e.d(t,{default:()=>c}),function(n,t,e,i,o){var a,s,c,d,l;for((o=n[i]=n[i]||{})._q=o._q||[],s=0,c=(a=["initialize","identify","updateOptions","pageLoad","track"]).length;s<c;++s)!function(n){o[n]=o[n]||function(){o._q[n===a[0]?"unshift":"push"]([n].concat([].slice.call(arguments,0)))}}(a[s]);(d=t.createElement(e)).async=!0,d.src="https://cdn.pendo.io/agent/static//pendo.js",(l=t.getElementsByTagName(e)[0]).parentNode.insertBefore(d,l)}(window,document,"script","pendo");const i="toolchain.dev.openshift.com/",o=`${i}sso-user-id`,a=`${i}sso-account-id`;let s=!1;const c=(n,t)=>{if("identify"===n){const{user:n}=t;if(n){const t=n.metadata.annotation?.[o],e=n.metadata.annotation?.[a];t&&window.pendo&&(s=!0,window.pendo[s?"identify":"initialize"]({visitor:{id:t},...e?{account:{id:e}}:null}))}}}}}]);
```

Comment:
Looks like some generated file? How did you generate it?

Reply:
@alexeykazakov these are generated from Christian's project at https://github.com/christianvogt/console-plugin-pendo.  It requires the nodejs tooling installed to build, I'm not sure if we want to introduce that kind of dependency to our build process but we can if required.  One way might be to include the actual JS/Typescript source and have Podman spin up a Node container to perform the build.

Reply:
Yes, I would like to have the plugin source code in our codeready-toolchain GitHub org.

I think the easiest option would be to move the actual JS/Typescript source here, in member-operator repo. Then add a new make target to generate the minimized JS out of it using nodejs.

And there are two options after that:
1. include that make target in our CI/CD targets so this min.js is always auto-generated (so we don't commit it to the repo)
2. keep this make target standalone and require developers to run the target manually every time the typescript source code is changed and commit it to GitHub. Disadvantage is that it's another manual step and it's hard to verify the proposed change/PR because the min.js is not readable (you have to trust the PR author that he/she did the generation properly) which makes some kind of e2e tests more critical in this case but it's also much harder to introduce any e2e tests with this option.

Reply:
No problem, I'll take care of that.  I think option 1 might be the best, simply because it doesn't have the disadvantages of option 2.